config/locales/devise.nl.yml

```diff
@@ -0,0 +1,63 @@
+# if me: DO NOT modify this file. if you need to add translations, add them to en.yml
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+nl:
+  devise:
+    confirmations:
+      confirmed: "Je emailadres is met succes bevestigd."
+      send_instructions: "Je zult een email ontvangen ter bevestiging en verdere instructies binnen enkele minuten."
+      send_paranoid_instructions: "Indien je emailadres reeds in onze database staat, zul je een email ontvangen met instructies om je account te activeren binnen enkele minuten."
+    failure:
+      already_authenticated: "Je bent reeds ingelegd."
```

Comment:
This should be "ingelogd", instead of "ingelegd".

Also wanted to note that "reeds" is pretty formal. It's not wrong, but "al" may be a bit better if you're also going to use informal "je".